packages/uui-button/lib/uui-button.element.ts

```diff
@@ -609,19 +608,17 @@ export class UUIButtonElement extends LabelMixin('', LitElement) {
     let element = html``;
     switch (this.state) {
       case 'waiting':
-        if (!customElements.get('uui-loader-circle')) {
-          console.warn(
-            'To properly render the waiting state, the uui-loader-circle element has to be registered'
-          );
-        }
+        demandCustomElement(this, 'uui-loader-circle');
         element = html`<uui-loader-circle id="loader"></uui-loader-circle>`;
         break;
       case 'success':
+        demandCustomElement(this, 'uui-icon');
```

Comment:
If the icon-registry is a required dependency why not also uui-icon instead of demandCustomElement?

Reply:
@iOvergaard maybe you can review your point on this.
But I think if we decide to move the Svgs out of the package then that properly should be another PR.

And if we are okay to keep them as part of the icon-registry-essential package then i would say this code is all good. :-)

Reply:
I agree - all good.